xml/html_title.xml

```diff
@@ -1838,22 +1838,22 @@
 
   <fingerprint pattern="^AVTECH Software, Inc\. - Room Alert(?:&amp;reg;)?\s+([a-zA-Z0-9/]+) - Advanced">
     <description>AVTECH RoomAlert</description>
-    <example hw.product="Room Alert 11E" hw.product.model="11E">AVTECH Software, Inc. - Room Alert 11E - Advanced Computer Room Environment Monitoring, Alerting &amp;amp; Automatic Corrective Action</example>
-    <example hw.product="Room Alert 32E/W" hw.product.model="32E/W">AVTECH Software, Inc. - Room Alert 32E/W - Advanced Computer Room Environment Monitoring, Alerting &amp;amp; Automatic Corrective Action</example>
-    <example hw.product="Room Alert 32E/W" hw.product.model="32E/W">AVTECH Software, Inc. - Room Alert&amp;reg; 32E/W - Advanced Computer Room Monitoring, Alerting &amp;amp; Automatic Corrective Action</example>
+    <example hw.product="Room Alert 11E" hw.model="11E">AVTECH Software, Inc. - Room Alert 11E - Advanced Computer Room Environment Monitoring, Alerting &amp;amp; Automatic Corrective Action</example>
+    <example hw.product="Room Alert 32E/W" hw.model="32E/W">AVTECH Software, Inc. - Room Alert 32E/W - Advanced Computer Room Environment Monitoring, Alerting &amp;amp; Automatic Corrective Action</example>
+    <example hw.product="Room Alert 32E/W" hw.model="32E/W">AVTECH Software, Inc. - Room Alert&amp;reg; 32E/W - Advanced Computer Room Monitoring, Alerting &amp;amp; Automatic Corrective Action</example>
     <param pos="0" name="hw.vendor" value="AVTECH"/>
     <param pos="0" name="hw.device" value="Monitoring"/>
-    <param pos="1" name="hw.product.model"/>
-    <param pos="0" name="hw.product" value="Room Alert {hw.product.model}"/>
+    <param pos="1" name="hw.model"/>
+    <param pos="0" name="hw.product" value="Room Alert {hw.model}"/>
   </fingerprint>
 
   <fingerprint pattern="^AVTECH Software, Inc\. - TemPageR\s+([a-zA-Z0-9/]+) - Real-Time">
     <description>AVTECH TepPageR</description>
-    <example hw.product="TemPageR 3E" hw.product.model="3E">AVTECH Software, Inc. - TemPageR 3E - Real-Time Temperature Monitor For IT &amp;amp; Facilities Environment Monitoring</example>
+    <example hw.product="TemPageR 3E" hw.model="3E">AVTECH Software, Inc. - TemPageR 3E - Real-Time Temperature Monitor For IT &amp;amp; Facilities Environment Monitoring</example>
     <param pos="0" name="hw.vendor" value="AVTECH"/>
     <param pos="0" name="hw.device" value="Monitoring"/>
-    <param pos="1" name="hw.product.model"/>
-    <param pos="0" name="hw.product" value="TemPageR {hw.product.model}"/>
+    <param pos="1" name="hw.model"/>
+    <param pos="0" name="hw.product" value="TemPageR {hw.model}"/>
```

Comment:
@hdm  - RE: These two fingerprints - you added these a while back.  I've changed `hw.product.model` to `hw.model` to be in keeping with the use of `hw.model` used elsewhere. Any concerns here?